torch/csrc/api/include/torch/mps.h

```diff
@@ -0,0 +1,21 @@
+#pragma once
```

Comment:
I am not clear on all these file paths. Are other backends following similar convention?

Reply:
- `torch/csrc/mps/Module.cpp`: counterpart to CUDA's `torch/csrc/cuda/Module.cpp`  to implement torch._C interface functions.
- `torch/csrc/api/include/torch/mps.h` and `torch/csrc/api/torch/mps.cpp` are C++ APIs (not python) for synchronize, manual_seed() and etc. Their cuda counterparts cuda.h and cuda.cpp are in the same folders as those files.
- `torch/mps/__init__.py` is the python interface to initialize the MPS module and implement python interfaces. It's cuda counterpart is in `torch/cuda/__init__.py`.